CONTRIBUTING.md

```diff
@@ -445,7 +447,7 @@ po2json -d superset -f jed1.x superset/translations/en/LC_MESSAGES/messages.po s
 If you get errors running `po2json`, you might be running the Ubuntu package with the same
 name, rather than the NodeJS package (they have a different format for the arguments). If
 there is a conflict, you may need to update your `PATH` environment variable or fully qualify
-the executable path (e.g. `/usr/local/bin/po2json` instead of `po2json`).
+the executable path (e.g. `/usr/local/bin/po2json` instead of `po2json`).If you get a lot of`[null,***]`in `messages.json` just delete all the `null,` ( eg:`"year":["年"]` is correct ,`"year":[null,"年"]`is incorrect)
```

Comment:
```suggestion
the executable path (e.g. `/usr/local/bin/po2json` instead of `po2json`).If you get a lot of`[null,***]`in `messages.json` just delete all the `null,` For example, `"year":["年"]` is correct while `"year":[null,"年"]`is incorrect
```

Reply:
You can copy-paste my comment above to `CONTRIBUTING.md` instead of trying to figure out what I changed.